src/pkg/cli/new.go

```diff
@@ -50,7 +50,17 @@ func FetchSamples(ctx context.Context) ([]Sample, error) {
 	return samples, err
 }
 
+// MixinFromSamples copies the sample files into the given directory, skipping existing files.
+func MixinFromSample(ctx context.Context, dir string, name string) error {
+	return copyFromSamples(ctx, dir, []string{name}, true)
+}
```

Comment:
How does this get used?

Reply:
The idea here is that we'd publish "mixin samples" that you could "add" to your existing project, for example Dockerfile or compose.yaml files, possible even "frontend" or "backend", etc.. It's related to https://github.com/DefangLabs/samples/pull/165 but I'm still thinking about the dev experience.